packages/react/src/Testimonial/Testimonial.module.css

```diff
@@ -17,12 +17,24 @@
   justify-items: center;
 }
 
+.Testimonial--size-large {
+  gap: var(--base-size-24);
+}
+
 .Testimonial-quote {
   font-family: var(--brand-fontStack-sansSerif);
+  font-size: var(--brand-text-size-300);
   margin-top: calc(var(--base-size-16) * -1); /* negate grid gap */
   display: block;
 }
 
+.Testimonial--size-large .Testimonial-quote {
+  font-size: var(--brand-text-size-700);
+  font-weight: var(--base-text-weight-semibold);
+  letter-spacing: -0.03em;
+  line-height: 1.2; /* requires custom line-height */
```

Comment:
should we use `rem` values here? 

Looks like the jump between `--brand-text-lineHeight-600` (2.75rem) to `--brand-text-lineHeight-700` (4rem) is a bit inconsistent, not sure if we should adapt  the `--brand-text-lineHeight-700` value at some point.

Reply:
Yeah, we'll need to review the line-height scale include a closer value.